py-polars/polars/config.py

```diff
@@ -20,14 +20,38 @@ def _get_float_precision() -> int:
     return -1
 
 
+def _get_digit_group_separator() -> str:  # pragma: no cover
+    return ","
+
+
+def _get_digit_group_size() -> int:
+    return 3
+
+
+def _get_decimal_separator() -> str:
+    return "."
+
+
+def _get_trim_decimal_zeros() -> bool:
+    return False
```

Comment:
I don't think these should be here?

Reply:
They don't now that I've integrated the `plr` suggestion  :)